docs/user/document/references-usage.en.md

```diff
@@ -1,11 +1,13 @@
 # References - Usage
 
+There should be one reference item that provides a canonical URL for this document, i.e. it has the category `self`, the `url` starts with `https://` and ends with a valid filename for the CSAF document according to the rules in section 5.1. of the specification.
+
 All links that have any relevance in the overall context of the document should be listed here.
-This could include for example original advisories or advisories of the manufacturer or especially the coordinator.
+This could include original advisories, advisories of the manufacturer or advisories of the coordinator.
 If other advisories (especially from suppliers) are referenced, they should be listed here.
 
 There should be one reference item that provides a canonical URL for this document, i.e. it has the category `self`, the `url` starts with `https://` and ends with a valid filename for the CSAF document according to the rules in section 5.1. of the specification.
 
-It is recommended to link the recommended cybersecurity best practices of the team where advisories are found.
+It is recommended to link the recommended cybersecurity best practices of the organization and to link a page where the advisory is found.
```

Comment:
```suggestion
It is recommended to include links to the cybersecurity best practices documents of the organization and the website which lists the advisories.

Reply:
As I couldn't think of a better wording I took over the suggestion